### PR TITLE
Updated what's new

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Updates the What's New topic with the latest changes.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html?

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,179 @@ description: |
...
+  type: Technical changes
+  date: October 31, 2019
+  link: https://github.com/magento/devdocs/pull/5883
+- description: Added the [Password Hashing](https://devdocs.magento.com/guides/v2.3/config-guide/secy/hashing.html)
+    topic to the _Configuration Guide_.
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: October 29, 2019
+  link: https://github.com/magento/devdocs/pull/5874
+- description: Added the following information to the [Elasticsearch set up](https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-elastic.html)<br/>topic
```
Comment (shrielenee) [inline _data/whats-new.yml]: Is this "set up" or "setup"? Seems like it might be one word.

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,179 @@ description: |
...
+  date: October 22, 2019
+  link: https://github.com/magento/devdocs/pull/5703
+- description: Updated the [Cloud launch modes](https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-config.html)  documentation
+    to configure Varnish as a http-cache-host so that Varnish purges with a Magento
+    cache clean.
+  versions: 2.x
+  type: Technical changes
+  date: October 22, 2019
+  link: https://github.com/magento/devdocs/pull/5721
+- description: Added the documentation for the [DynamicRowsRecord](https://devdocs.magento.com/guides/v2.2/ui_comp_guide/components/ui-dynamicrows-record.html)
```
Comment (shrielenee) [inline _data/whats-new.yml]: Should DynamicRowsRecord be in code formatting?

File: _data/whats-new.yml [hunk 3]
```diff
@@ -4,8 +4,179 @@ description: |
...
+  type: Major update
+  date: October 18, 2019
+  link: https://github.com/magento/devdocs/pull/5800
+- description: Moved the `CategoryInterface` attributes to a separate [topic](https://devdocs.magento.com/guides/v2.3/graphql/product/category-interface.html)
+    in the _GraphQL Developer Guide_.
+  versions: 2.3.x
+  type: New topic
+  date: October 16, 2019
+  link: https://github.com/magento/devdocs/pull/5718
+- description: Added an "Errors" section to the [changeCustomerPassword mutation](https://devdocs.magento.com/guides/v2.3/graphql/mutations/change-customer-password.html)
```
Comment (shrielenee) [inline _data/whats-new.yml]: I am not sure you reference doc sections in other line items in quotes... check for consistency.

Comment (shrielenee) [reply _data/whats-new.yml]: Also consistency in ending punctuation.

File: _data/whats-new.yml [hunk 4]
```diff
@@ -4,8 +4,179 @@ description: |
...
+  type: Technical changes
+  date: October 14, 2019
+  link: https://github.com/magento/devdocs/pull/5565
+- description: Added [Backward incompatible changes reference](https://devdocs.magento.com/guides/v2.3/release-notes/backward-incompatible-changes/reference.html)
+    for the Magento 2.3.3 release.
+  versions: 2.3.3
+  type: Major update
+  date: October 14, 2019
+  link: https://github.com/magento/devdocs/pull/5686
+- description: Added a warning to ensure that the [CRYPT_KEY](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#crypt_key)  deploy
```
Comment (shrielenee) [inline _data/whats-new.yml]: CRYPT_KEY in code format?

File: _data/whats-new.yml [hunk 5]
```diff
@@ -4,8 +4,179 @@ description: |
...
+  link: https://github.com/magento/devdocs/pull/5641
+- description: Updated the _Command-line tools reference_:<br/>- Regenerated [bin/magento
+    (Open Source)](https://devdocs.magento.com/guides/v2.3/reference/cli/magento.html)<br/>-
+    Added [bin/magento (Commerce)](https://devdocs.magento.com/guides/v2.3/reference/cli/magento-commerce.html)<br/>-
+    Added [magento-cloud]( https://devdocs.magento.com/guides/v2.3/reference/cli/magento-cloud.html)
+  versions: 2.2.10, 2.3.3
+  type: New topic
+  date: October 10, 2019
+  link: https://github.com/magento/devdocs/pull/5642
+- description: Amazon Sales Channel version 3.0.0 is available in the [Marketplace](https://marketplace.magento.com/magento-module-amazon.html).
```
Comment (shrielenee) [inline _data/whats-new.yml]: This doesn't seem like an action statement? Like "Added release notes for the Amazon Sales Channel version 3.0.0, which is now available in the Marketplace" or similar.

Comment (meker12) [inline _data/whats-new.yml]:
```suggestion
- description: Added Amazon Sales Channel version 3.0.0 [Release Notes](https://devdocs.magento.com/extensions/amazon-sales/release-notes/).
```

File: _data/whats-new.yml [hunk 6]
```diff
@@ -4,8 +4,179 @@ description: |
...
+    Check out the [Release Notes](https://devdocs.magento.com/extensions/amazon-sales/release-notes/).
```
Comment (meker12) [inline _data/whats-new.yml]:
```suggestion
```

**Comment by meker12** [comment]
running tests

**Pull request merged**